src/main/webapp/WEB-INF/view/activityfeed.jsp

```diff
@@ -1,26 +1,75 @@
+<%@ page import="java.util.List" %>
+<%@ page import="codeu.model.data.Activity" %>
+<%@ page import="codeu.model.store.basic.ActivityStore" %>
+<%@ page import="codeu.model.store.basic.UserStore" %>
+
+<%
+List<Activity> activities = (List<Activity>) request.getAttribute("activities");    
+%>
 <!DOCTYPE>
 <html>
-	<head>
-	  <link rel="stylesheet" href="/css/main.css" type="text/css">
+  <head>
+    <link rel="stylesheet" href="/css/main.css" type="text/css">
+
+  </head>
+  
+  <body>
+    <nav>
+      <a id="navTitle" href="/">Trill</a>
+      <a href="/conversations">Conversations</a>
+        <% if (request.getSession().getAttribute("user") != null) { %>
+      <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+      <% } else { %>
+        <a href="/login">Login</a>
+      <% } %>
+      <!-- Add login checking for activity feed here -->
+      <a href="/activityfeed">Activity Feed</a>
+      <a href="/about.jsp">About</a>
+    </nav>
+    
+    <div id="container">
```

Comment:
I know you said you were looking to add a more "flowy" GUI when listing out the activities. Take a look at the section that displays messages in chat.jsp, it creates styling to the div that displays the chat so that it flows and allows users to scroll through it. I used it as a reference for how I display a users messages in the profile page.

Reply:
Alright, thanks!